clients/venice-admin-tool/src/main/java/com/linkedin/venice/Command.java

```diff
@@ -474,6 +363,11 @@ public enum Command {
       "dump-ingestion-state",
       "Dump the real-time ingestion state for a certain store version in a certain storage node",
       new Arg[] { SERVER_URL, STORE, VERSION }, new Arg[] { PARTITION }
+  ),
+  ADD_CHANGE_CAPTURE_VIEW(
```

Comment:
Should we also have a command for adding a view of any name? Or maybe it doesn't matter for now since we have no other view implementations...

Reply:
Yeah can add that.

We also need to add a patch to the controller for removing a view.  That currently doesn't work as intended :(